doc/source/train/examples/intel_gaudi/llama_pretrain.ipynb

```diff
@@ -0,0 +1,568 @@
+{
```

Comment:
High-level question: seems that it's using deepspeed zero-3 for pre-training. why we also include megatron here?

Reply:
this example import megatron just for data processing

Reply:
Hi, I removed the megatron dependency here, just using deepspeed to pre-train model.